public/app/features/dimensions/editors/ResourcePicker.tsx

```diff
@@ -33,12 +34,17 @@ interface ResourceItem {
 }
 
 export function ResourcePicker(props: Props) {
-  const { value, onChange, mediaType } = props;
-  const folders = (mediaType === 'icon' ? ['img/icons/unicons', 'img/icons/iot'] : ['img/bg']).map((v) => ({
-    label: v,
-    value: v,
-  }));
-  const folderOfCurrentValue = value ? folders.filter((folder) => value.indexOf(folder.value) > -1)[0] : folders[0];
+  const { value, onChange, mediaType, folderIndex } = props;
+  const folders = (mediaType === 'icon' ? ['img/icons/unicons', 'img/icons/iot', 'img/icons/geo'] : ['img/bg']).map(
```

Comment:
nit: add these to a constant.ts file to make it easier to change moving forward?